src/utils/images.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Util method to retrieve image details from /images folder from the relative file path,
+ * e.g. "/images/album%/picture$.jpg" -> { imageDirectory: "images%2Falbum%24", fileName: "picture%24.jpg" }
```

Comment:
Minor mistake in the comment. `imageDirectory` should be `images%2Falbum%25`. Note `%` is encoded to `%25` instead of `%24`
```suggestion
 * e.g. "/images/album%/picture$.jpg" -> { imageDirectory: "images%2Falbum%25", fileName: "picture%24.jpg" }
```